analyzers/rspec/cs/S6802.html

```diff
@@ -5,8 +5,13 @@ <h2>Why is this an issue?</h2>
 elements.</p>
 <p>The reason behind this is that Blazor rebuilds all lambda expressions within the loop every time the UI elements are rendered.</p>
 <h2>How to fix it</h2>
-<p>Ensure to not use a delegate in elements rendered in loops by using a collection of objects containing the delegate as an <a
-href="https://learn.microsoft.com/en-us/dotnet/api/system.action">Action</a>.</p>
+<p>Ensure to not use a delegate in elements rendered in loops by either:</p>
```

Comment:
Note: the update of `sonar-dotnet`, after the comments in https://github.com/SonarSource/rspec/pull/3452 are addressed, can be done directly during release. I have added a remark in the description of the issue to [Update RSPEC for 9.14](https://github.com/SonarSource/sonar-dotnet/issues/8290).

Reply:
I already did the update.